transform/views/test_views.py

```diff
@@ -216,6 +213,23 @@ def images_test():
         return send_file(zipfile, mimetype='application/zip')
 
 
+@app.route('/pdf-test', methods=['GET'])
```

Comment:
Again, I'm not sure having test_routes in a production app is a good idea. Just more scope for errors.